ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1093,11 +1093,24 @@ void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
             break;
         }
 
-        // ensure type_mask specifies to use attitude and thrust
-        if ((packet.type_mask & ((1<<7)|(1<<6))) != 0) {
+        const bool roll_rate_ignore   = packet.type_mask & ATTITUDE_TARGET_TYPEMASK_BODY_ROLL_RATE_IGNORE;
```

Comment:
I wonder if this code would read better if these were inverted so
```suggestion
        const bool use_roll_rate    = !(packet.type_mask & ATTITUDE_TARGET_TYPEMASK_BODY_ROLL_RATE_IGNORE);
```

... there's a bunch of double-negatives in this function that would disappear further down with this sort of change.

Reply:
This is the same pattern we have used right below this in `MAVLINK_MSG_ID_SET_POSITION_TARGET_LOCAL_NED` which  matches to what the mavlink type_mask is which is an ignore_mask not a use_mask.

If we want to change this then lets change it everywhere at once afterwards?